core/src/main/java/nl/nn/adapterframework/jdbc/JdbcQuerySenderBase.java

```diff
@@ -119,6 +120,8 @@
 	protected String[] columnsReturnedList=null;
 	private boolean streamResultToServlet=false;
 
+	private String sqlDialect = AppConstants.getInstance().getString("jdbc.sqlDialect", null);
```

Comment:
Ik denk dat het zinvol is om ook een getter en setter voor sqlDialect te maken, zodat je de waarde ook voor een individuele sender kan zetten. Dan kan je bijvoorbeeld in verschillende modules verschillende dialecten gebruiken.

Reply:
https://github.com/ibissource/iaf/pull/251/commits/9cc34b0e94fdd62ca404258bf4f056dcaa9717ae